rustls-libssl/src/cache.rs

```diff
@@ -5,9 +5,13 @@ use std::time::SystemTime;
 
 use rustls::client::ClientSessionMemoryCache;
 use rustls::client::ClientSessionStore;
+use rustls::server::StoresServerSessions;
 
-use crate::entry::{SSL_CTX_new_session_cb, SSL_CTX_sess_get_cb, SSL_CTX_sess_remove_cb, SSL_CTX};
-use crate::SslSession;
+use crate::callbacks;
+use crate::entry::{
+    SSL_CTX_new_session_cb, SSL_CTX_sess_get_cb, SSL_CTX_sess_remove_cb, SSL_CTX, SSL_SESSION,
+};
+use crate::{SslSession, SslSessionLookup};
```

Comment:
nit: I think the import of `callbacks` on L10 could be folded into the other top-level crate imports on L14:
`use crate::{callbacks, SslSession, SslSessionLookup};`.